internal/pkg/service/stream/storage/statistics/provider_test.go

```diff
@@ -372,6 +373,9 @@ func TestCaches(t *testing.T) {
 			expectedRevision = header.Revision
 		}
 
+		// Test repository
+		tc.Assert(repo)
+
```

Comment:
Missing test for repository (it implements the provider interface).

I moved test one level upper.